packages/typespec-ts/test/modularUnit/scenarios/classicClient/classicClient.md

```diff
@@ -25,6 +25,8 @@ enum Versions {
   @useDependency(Azure.Core.Versions.v1_0_Preview_2)
   `2022-08-30`,
 }
+
+op foo(): void;
```

Comment:
Could you create an issue to support empty model generation for modular?

Reply:
#3177